apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/CommitResponse.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Google.Cloud.Spanner.Data
+{
+    /// <summary>
+    /// Immutable class representing a commit response.
+    /// </summary>
+    public sealed class CommitResponse
+    {
+        private const string MISSING_COMMIT_STATS = "The transaction did not return any CommitStats. Set SpannerConnection.ReturnCommitStats=true to request commit statistics.";
+
+        internal CommitResponse(V1.CommitResponse proto)
+        {
+            Proto = GaxPreconditions.CheckNotNull(proto, nameof(proto));
+        }
+
+        /// <summary>
+        /// The proto representation of the commit response. Must not be mutated
+        /// or exposed publicly.
+        /// </summary>
+        internal V1.CommitResponse Proto { get; }
+
+        /// <summary>
+        /// The commit timestamp of the transaction.
+        /// </summary>
+        public DateTime CommitTimestamp { get => Proto.CommitTimestamp.ToDateTime(); }
+
+        /// <summary>
```

Comment:
If the intention of the blank lines is to signify multiple paragraphs, it would be best to use `<para>` tags. I suspect that only the first paragraph belongs in `summary` though, and the rest should be in `remarks`.